packages/web/docs/public/apollo-router-download.sh

```diff
@@ -11,7 +11,7 @@ parse_args() {
     key="$1"
 
     case $key in
-      -c | --commit)
+      -c | --commit | -v | --version)
```

Comment:
@kamilkisiela can you confirm it's going to work? 😆

Reply:
you can use previews to verify it